pkg/resources/deployment.go

```diff
@@ -0,0 +1,74 @@
+package resources
+
+import (
+	appsv1beta1 "k8s.io/api/apps/v1beta1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func Deployment(namespace string) *appsv1beta1.Deployment {
+	return &appsv1beta1.Deployment{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "ark",
+			Namespace: namespace,
+		},
+		Spec: appsv1beta1.DeploymentSpec{
+			Template: corev1.PodTemplateSpec{
+				ObjectMeta: metav1.ObjectMeta{
+					Labels: map[string]string{
+						"component": "ark",
+					},
+				},
+				Spec: corev1.PodSpec{
+					RestartPolicy:      corev1.RestartPolicyAlways,
+					ServiceAccountName: "ark",
+					Containers: []corev1.Container{
+						{
+							Name:            "ark",
+							Image:           "gcr.io/heptio-images/ark:latest",
+							ImagePullPolicy: corev1.PullAlways,
```

Comment:
I usually prefer PullIfNotPresent. WDYT?

Reply:
I always pull if not present, but the yaml hard-codes the latest tag and we can't pull if not present with latest. How about we parameterize the tag and `if tag == "latest"`, we set the `ImagePullPolicy` to pull always and use if not present otherwise?

Reply:
SGTM